specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -845,14 +840,14 @@ these SPIs and policies.
 This specification defines the mapping of
 application components to industry-standard network protocols. The
 mapping allows client access to the application components from systems
-that have not installed Java EE product technology. See
+that have not installed Jakarta EE product technology. See
 link:#a2845[See Interoperability],” for
 details on the network protocol support required for interoperability.
 
-The Java EE Product Provider is required to
+The Jakarta EE Product Provider is required to
 publish the installed application components on the industry-standard
-protocols. This specification defines the mapping of servlets and JSP
-pages to the HTTP and HTTPS protocols, and the mapping of EJB components
+protocols. This specification defines the mapping of Jakarta Servlets and Jakarta Server Pages
```

Comment:
"Jakarta Servlets" should just be "servlets" since it's a reference to the application
components, not the specification.

Reply:
In the same light, then we should probably just change the JSP reference to "server pages" (all lowercase).  And, "enterprise beans" in the next sentence.  Agree, @bshannon?

Reply:
@kwsutter  I worry that "server pages" isn't obviously a reference to JSP pages,
it feels like a very "generic" term, although it does read better than
"Jakarta Server Pages pages".  :-)

Reply:
Yeah, that's what I was thinking as well...  Let's go with readability of "server pages" for now.  We can change later if there is confusion.